packages/react-dev-utils/launchEditor.js

```diff
@@ -12,6 +12,7 @@ const child_process = require('child_process');
 const os = require('os');
 const chalk = require('chalk');
 const shellQuote = require('shell-quote');
+const isWsl = require('is-wsl');
```

Comment:
Is there a way to do this without a package?

It looks like it's a small amount of code - https://github.com/sindresorhus/is-wsl/blob/master/index.js

Which seems to align with this:
https://github.com/microsoft/WSL/issues/423#issuecomment-221627364

Reply:
Sure. There is no need for this package.
I just added it because I saw also `is-root` to be used which is a similar small package. That gave me the expression that adding small packages like this is okay.
I'll remove it.